src/c++/perf_analyzer/genai-perf/README.md

```diff
@@ -62,7 +62,7 @@ Available starting with the 24.03 release of the
 [Triton Server SDK container](https://ngc.nvidia.com/catalog/containers/nvidia:tritonserver).
 
 ```bash
-RELEASE="24.03"
+RELEASE="24.04"
```

Comment:
@dyastremsky  can we get rid of this practice. 
May be instead of dedicated version we can mention `YY.MM`  and leave it unchanged forever?
```suggestion
RELEASE="<YY.MM>" # please refer to Triton Inference Server latest release  
```

Reply:
Sure, that sounds good to me.